website/search/elastic_search.py

```diff
@@ -419,6 +424,48 @@ def bulk_update_nodes(serialize, nodes, index=None):
     if actions:
         return helpers.bulk(client(), actions)
 
+def serialize_cgm_contributor(contrib):
+    return {
+        'fullname': contrib['fullname'],
+        'url': '/{}/'.format(contrib['guids___id']) if contrib['is_active'] else None
+    }
+
+def serialize_cgm(cgm):
+    obj = cgm.guid.referent
+    contributors = []
+    if hasattr(obj, '_contributors'):
+        contributors = obj._contributors.filter(contributor__visible=True).order_by('contributor___order').values('fullname', 'guids___id', 'is_active')
+
+    return {
+        'id': cgm._id,
+        'abstract': getattr(obj, 'description', ''),
+        'collectedType': getattr(cgm, 'collected_type'),
+        'contributors': [serialize_cgm_contributor(contrib) for contrib in contributors],
+        'status': cgm.status,
+        'subjects': list(cgm.subjects.values_list('text', flat=True)),
+        'title': getattr(obj, 'title'),
```

Comment:
Minor: These will throw `AttributeError`s for objects that don't have a `title` or `url`. This matters little now, but will whenever we allow objects other than nodes to be collected.